### DISCUSSION TITLE
Image preprocessing acceleration

### DISCUSSION THREAD

**Discussion by daohu527** [daohu527/Dig-into-Apollo#55] [Ideas]
Here is a collection of some image preprocessing implementations, such as resize, crop, etc.

## opencv
https://docs.opencv.org/4.x/da/d54/group__imgproc__transform.html

## cuda
https://cvcuda.github.io/samples/cpp_samples/cropresize.html

## torch
torchvison.resize

**Comment by 20141314** [comment]
这是来自QQ邮箱的自动回复邮件。
